src/gui/wxpython/wx.metadata/db.csw.admin/db.csw.admin.py

```diff
@@ -283,7 +283,7 @@ def run(self, argv):
 
         if self.COMMAND not in ["post_xml"]:
             SCP = configparser.SafeConfigParser()
```

Comment:
```python
>>> SCP = configparser.SafeConfigParser()
<stdin>:1: DeprecationWarning: The SafeConfigParser class has been renamed to ConfigParser in Python 3.2. This alias will be removed in Python 3.12. Use ConfigParser directly instead.
```

Reply:
I would recommend to change this in another commit. Having to separate ones makes it easier to find reasoning behind commits through `git blame`

Reply:
Ok.

Reply:
@tmszi: Funny thing - I apparently created a PR fixing the issue you had mentioned already more than a month ago as #919, but forgot about that. Going to merge that one too.